build.rs

```diff
@@ -7,78 +7,66 @@ use std::{
     str,
 };
 
-#[cfg(not(target_os = "macos"))]
-const LINUX_SDK_URL: &str  = "https://github.com/jabber-tools/cognitive-services-speech-sdk-rs-files/blob/main/SpeechSDK/1.22.0/linux/SpeechSDK-Linux-1.22.0.tar.gz?raw=true";
+const SPEECH_SDK_VERSION: &str = "1.36.0";
 
-#[cfg(not(target_os = "macos"))]
 fn download_file(url: &str, dst: &str) {
     Command::new("curl")
-        .args(&["-SL", url, "-o", dst])
+        .args(["-SL", url, "-o", dst])
         .status()
         .expect("failed to download Speech SDK!");
 }
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(target_os = "linux")]
 fn main() {
-    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
-    // copying SpeechSDK from local folder just worked fine but crates.io allows
```

Comment:
ok now I remember local files were here but this did not work with crates.io during package publishing. Still I would like to keep this commented section here as it might give you hint how to use local speech sdk if needed

Reply:
>  Still I would like to keep this commented section here as it might give you hint how to use local speech sdk if needed

Sure.